content/en/user-guide/aws/s3/index.md

```diff
@@ -89,3 +87,77 @@ S3_DIRS=/tmp/s3-buckets/first-bucket:my-first-bucket,/tmp/s3-buckets/second-buck
 ```
 
 In both configurations of `S3_DIRS`, if Localstack is started and the path(s) specified in `S3_DIRS` are not empty, the S3 buckets will be pre-populated with files.
+
+## Configuring Cross-Origin Resource Sharing on S3
+
+You can configure Cross-Origin Resource Sharing (CORS) on a LocalStack S3 bucket using AWS Command Line Interface (CLI). It would allow your local application to communicate directly with an S3 bucket in LocalStack. By default, LocalStack will apply specific CORS rules to all requests to allow you to display and access your resources through [LocalStack Web Application](https://app.localstack.cloud). If no CORS rules are configured for your S3 bucket, LocalStack will apply default rules unless specified otherwise.
+
+To configure CORS rules for your S3 bucket, you need to have the `awslocal` wrapper installed and configured. Optionally, you can run a local web application on [localhost:3000](http://localhost:3000). Run the following command on your terminal to create your S3 bucket:
```

Comment:
question: people can also use the SDK or anything: should we say "to follow this tutorial/sample" instead of "to configure CORS rules"? Not sure if I make sense.